Common/UnitDefinitions/Power.json

```diff
@@ -94,6 +94,19 @@
           "Abbreviations": [ "Btu/hr" ]
         }
       ]
+    },
+    {
+      "SingularName": "SolarLuminosity",
+      "PluralName": "SolarLuminosities",
+      "BaseUnit": "Watt",
```

Comment:
I think this is the culprit of your unit test errors. Units should not have `"BaseUnit"` defined, only quantities, so you can remove this.